imagery/i.rectify/main.c

```diff
@@ -151,8 +153,14 @@ int main(int argc, char *argv[])
     interpolate = menu[method].method;
 
     G_strip(grp->answer);
-    strcpy(group.name, grp->answer);
-    strcpy(extension, ext->answer);
+    len = G_strlcpy(group.name, grp->answer, sizeof(group.name));
+    if (len >= sizeof(group.name)) {
+        G_fatal_error(_("Group name <%s> is too long"), grp->answer);
+    }
+    len = G_strlcpy(extension, ext->answer, sizeof(extension));
+    if (len >= sizeof(extension)) {
+        G_fatal_error(_("Extension <%s> is too long"), ext->answer);
+    }
```

Comment:
It will break, it is not a catastrophe in itself, in this case readability is still good.

In #4087, for example, I recommended to use a len variable because the if statements were already indented, breaking the if condition into two lines would have caused less readable code in that case.

Reply:
Ok then i will include it in if statement and remove the variable